File: app/src/main/java/net/osdn/gokigen/blecontrol/lib/ui/settings/bluetooth/SettingsBluetoothViewModel.java
package net.osdn.gokigen.blecontrol.lib.ui.settings.bluetooth;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class SettingsBluetoothViewModel extends ViewModel
{
    private MutableLiveData<String> mText;

    public SettingsBluetoothViewModel()
    {
        mText = new MutableLiveData<>();
        mText.setValue("Bluetooth Connection Test");
    }

    public LiveData<String> getText() {
        return mText;
    }
}

File: app/src/main/java/net/osdn/gokigen/blecontrol/lib/ui/settings/bluetooth/SettingsBluetoothViewModel.kt
package net.osdn.gokigen.blecontrol.lib.ui.settings.bluetooth

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class SettingsBluetoothViewModel : ViewModel() {
    private val mText: MutableLiveData<String?>?

    init {
        mText = MutableLiveData<String?>()
        mText.setValue("Bluetooth Connection Test")
    }

    val text: LiveData<String?>?
        get() = mText
}